wavelog/docker-compose.yml

```
@@ -0,0 +1,30 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: wavelog_app_1
+      APP_PORT: 80
+      PROXY_AUTH_ADD: "false"
+
+  app:
+    image: ghcr.io/wavelog/wavelog:2.0.7@sha256:40721405cfbcbbb1e861c1713202f69cb3416b2753ed5cce02e5542235245eb4
```

why making your life harder


```suggestion
    image: ghcr.io/wavelog/wavelog:latest
```

---

This is an Umbrel safety measure.

---

well okay :-D